server/webapp/static/backup_in_progress.html

```diff
@@ -1,6 +1,6 @@
 ﻿<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
Consider moving this file out of `/static` because it'll be reachable if users connect to that URL directly.